.github/workflows/e2e-test-ci.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         run: |
           go get -u github.com/onsi/ginkgo/ginkgo
           sudo cp ~/go/bin/ginkgo /usr/local/bin
+      - name: Create K8s cluster
+        run: |
+          make kind-up
+          kubectl get nodes
```

Comment:
should we need add `kubectl wait --for=condition=Ready nodes --all` to wait node ready ?

Reply:
Good point! Changed.